appendix-basic65.tex

```diff
@@ -8436,7 +8559,7 @@ \subsection{RPALETTE}
 \begin{description}[leftmargin=2cm,style=nextline]
 \item [Token:] \$CE \$0D
 \item [Format:] {\bf RPALETTE(}screen{\bf,} index{\bf,} rgb{\bf)}
-\item [Usage:]  Returns the red, green or blue value of
+\item [Usage:]  The red, green, or blue value of
```

Comment:
Don't believe the ',' before the 'or' is necessary.

Reply:
per Oxford comma remark, above